github_actions/spec/dependabot/github_actions/file_fetcher_spec.rb

```diff
@@ -31,13 +31,20 @@
 
   before { allow(file_fetcher_instance).to receive(:commit).and_return("sha") }
 
-  context "with a workflow file" do
+  context "with workflow files" do
     before do
+      stub_request(:get, url + "?ref=sha").
+        with(headers: { "Authorization" => "token token" }).
+        to_return(
+          status: 200,
+          body: fixture("github", "contents_githubaction_repo_base_basic.json"),
+          headers: { "content-type" => "application/json" }
+        )
```

Comment:
Why is this change important for adding composite actions?

Reply:
The code is now looking for files in the root of the directory, and an error is returned by the mocking library in case a "request" is made to this URL and nothing is there to respond. 
Using an actual example of how a root of a repository would look like was just bonus.

Reply:
Perfect, thank you!